client/src/modules/patients/registry/registry.js

```diff
@@ -193,15 +185,6 @@ function PatientRegistryController($state, Patients, Notify, AppCache,
 
   // startup function. Checks for cached filters and loads them.  This behavior could be changed.
   function startup() {
-
-    if ($state.params.filters) {
```

Comment:
You still need to do something like this -- it will break the links from the Payments and Invoice Registries into the Patient Registry.

I suggest you implement something like https://github.com/IMA-WorldHealth/bhima-2.X/issues/1935 to solve this.